jenkins/databricks/build.sh

```diff
@@ -20,9 +20,9 @@ set -ex
 SPARKSRCTGZ=$1
 # version of Apache Spark we are building against
 BASE_SPARK_VERSION=$2
-BUILD_PROFILES=$3
+MVN_OPT=$3
```

Comment:
Additional options for mvn build on Databricks, e.g. MVN_OPT=`-Dspark-rapids-jnv.version=20220516.081920.30`